test/e2e/reaper_test.go

```diff
@@ -122,18 +130,18 @@ func createMultiReaper(t *testing.T, ctx context.Context, namespace string, f *f
 }
 
 func createReaperAndDatacenter(t *testing.T, ctx context.Context, namespace string, f *framework.E2eFramework) {
-
+	clusterName := "test"
 	dcKey := framework.ClusterKey{K8sContext: "kind-k8ssandra-0", NamespacedName: types.NamespacedName{Namespace: namespace, Name: "dc1"}}
 	reaperKey := framework.ClusterKey{K8sContext: "kind-k8ssandra-0", NamespacedName: types.NamespacedName{Namespace: namespace, Name: "reaper1"}}
 
 	checkDatacenterReady(t, ctx, dcKey, f)
 
 	t.Log("create Reaper keyspace")
-	_, err := f.ExecuteCql(ctx, "kind-k8ssandra-0", namespace, "test", "test-dc1-rack1-sts-0",
+	_, err := f.ExecuteCql(ctx, "kind-k8ssandra-0", namespace, clusterName, fmt.Sprintf("%s-dc1-rack1-sts-0", clusterName),
```

Comment:
Why this change? The code looks less obvious to me now compared to the previous version.